externals/CMakeLists.txt

```diff
@@ -86,7 +110,7 @@ set(${COMPONENT}_LINKER_LANGUAGE C)
 if(MLCPSIMPLEX_FOUND) 
   set(${COMPONENT}_LINKER_LANGUAGE CXX)
 endif()
-set(${COMPONENT}_LINK_LIBRARIES ${SICONOS_LINK_LIBRARIES})
+set(${COMPONENT}_LINK_LIBRARIES ${SICONOS_LINK_LIBRARIES} ${CXSparse_LIBRARY})
 
```

Comment:
This doesn't look right. The library should be added to SICONOS_LINK_LIBRARIES. I believe this happens in the compile_with macro.

Reply:
Yeah, please review this heavily ;)  I'll take a look.

Reply:
I'll move SuiteSparse configuration stuff to `siconos_find_common_dep`.